assets/models/system/distilgpt2/spec.yaml

```diff
@@ -114,4 +119,20 @@ tags:
       Standard_ND96amsr_A100_v4,
       Standard_ND40rs_v2
     ]
-version: 12
+  finetune_compute_allow_list:
+    [
+      Standard_NV12s_v3,
+      Standard_NV24s_v3,
+      Standard_NV48s_v3,
+      Standard_NC6s_v3,
+      Standard_NC12s_v3,
+      Standard_NC24s_v3,
+      Standard_NC24rs_v3,
+      Standard_NC4as_T4_v3,
+      Standard_NC8as_T4_v3,
+      Standard_NC16as_T4_v3,
+      Standard_NC64as_T4_v3,
+      Standard_ND40rs_v2,
+      Standard_ND96asr_v4,
+    ]
+version: 13
```

Comment:
Can you check the version on the repo and then increment accordingly please?

Reply:
Fixed